FIRO_TSEnsembles/src/main/java/hec/stats/TotalFlow.java

```diff
@@ -1,13 +1,13 @@
 package hec.stats;
 
-public class CumulativeFlow implements Computable, Configurable {
+public class TotalFlow implements Computable, Configurable {
```

Comment:
is it flow? or just total of the array of data?

Reply:
Can be more generic so Total of the array